### DISCUSSION TITLE
Error while running the frontend ansible-playbook  script

### DISCUSSION THREAD

**Discussion by bharath-gaveni** [learndevopsonline/learndevopsonline#1600] [Q&A] [answered]
Error while running the frontend ansible-playbook  script using role_name.

![Screenshot (3)](https://github.com/learndevopsonline/learndevopsonline/assets/149563378/0c173402-9eeb-409a-ba26-099f8ccd8563)

**Comment by sreedharm07** [comment, marked as answer]
Hi @bharath-gaveni 
your frontend was out of space please change instance type
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/60710281-6b81-4025-af71-4af442f29312)

**Comment by bharath-gaveni** [comment]
hi @sreedharm07 
Error got resolved now its working fine, thank you for your quick response
